File: VB/Helpers/MyScrollBarViewInfo.vb
Imports Microsoft.VisualBasic
Imports DevExpress.XtraEditors
Imports DevExpress.XtraEditors.ViewInfo
Imports System
Imports System.Drawing

Namespace WindowsApplication1
	Public Class MyScrollBarViewInfo
		Inherits ScrollBarViewInfo
		Private _ScrollBarOffset As Point
		Public Property ScrollBarOffset() As Point
			Get
				Return _ScrollBarOffset
			End Get
			Set(ByVal value As Point)
				_ScrollBarOffset = value
			End Set
		End Property
		Public Sub New(ByVal scrollBar As IScrollBar)
			MyBase.New(scrollBar)
		End Sub
		Protected Overrides ReadOnly Property CalcThumbButtonBounds() As Rectangle
			Get
				Dim bounds As Rectangle = MyBase.CalcThumbButtonBounds
				bounds.Offset(ScrollBarOffset)
				Return bounds
			End Get
		End Property
		Public Overrides ReadOnly Property VisibleDecButtonBounds() As Rectangle
			Get
				Dim rect As Rectangle = MyBase.VisibleDecButtonBounds
				rect.Offset(ScrollBarOffset)
				Return rect
			End Get
		End Property
		Public Overrides ReadOnly Property VisibleIncButtonBounds() As Rectangle
			Get
				Dim rect As Rectangle = MyBase.VisibleIncButtonBounds
				rect.Offset(ScrollBarOffset)
				Return rect
			End Get
		End Property
	End Class
End Namespace

File: CS/Helpers/MyScrollBarViewInfo.cs
using DevExpress.XtraEditors;
using DevExpress.XtraEditors.ViewInfo;
using System;
using System.Drawing;

namespace WindowsApplication1
{
    public class MyScrollBarViewInfo : ScrollBarViewInfo
    {
        private Point _ScrollBarOffset;
        public Point ScrollBarOffset
        {
            get
            {
                return _ScrollBarOffset;
            }
            set
            {
                _ScrollBarOffset = value;
            }
        }
        public MyScrollBarViewInfo(IScrollBar scrollBar)
            : base(scrollBar)
        {
        }
        protected override Rectangle CalcThumbButtonBounds
        {
            get
            {
                Rectangle bounds = base.CalcThumbButtonBounds;
                bounds.Offset(ScrollBarOffset);
                return bounds;
            }
        }
        public override Rectangle VisibleDecButtonBounds
        {
            get
            {
                Rectangle rect = base.VisibleDecButtonBounds;
                rect.Offset(ScrollBarOffset);
                return rect;
            }
        }
        public override Rectangle VisibleIncButtonBounds
        {
            get
            {
                Rectangle rect = base.VisibleIncButtonBounds;
                rect.Offset(ScrollBarOffset);
                return rect;
            }
        }
    }
}
